src/components/RenderRouter/ListItem.scss

```diff
@@ -30,13 +30,43 @@
   }
 }
 
+.ListInstagramButton{
+  &:hover{
+    text-emphasis: none;
+    color:$onyxblacktext;
+  }
+  display: block;
+  min-width: 200px;
+  width: 8vw;
+  border: solid 3px #1a1a1a;
+  box-sizing: border-box;
+  padding: 16px;
+  text-decoration: none !important;
+  
+  margin-right:3px;
+  font-family: Graphik Web;
+  font-weight: bold;
+  margin-top: 10px;
+  margin-left:20vw;
+}
 @media (min-width: 768px) {
   .ListItemEventButton {
     background-color: #ffffff00 !important;
     color: $onyxblacktext !important;
     border: 0px !important;
   }
 
+  .ListInstagramContainer{
+    left:20vw;
```

Comment:
Maybe use CSS grid so that the layout follows the design on Figma.

On Figma, the only grid patterns that show up are 2x4 and 4x2.